canonical-web-and-design/maas-ui
# Add dashboard configuration form

## Done

- Creates form for enabling and disabling discoverable subnets.

## QA

- Go to [/dashboard/configuration](http://0.0.0.0:8400/MAAS/r/dashboard/configuration)
- Check that after toggling and saving subnet's the same is change can be seen at http://bolla.internal:5240/MAAS/l/dashboard

## Fixes

Fixes: https://github.com/canonical-web-and-design/app-squad/issues/72
